### PR TITLE
Monitor your Dedicated cluster

### PR DESCRIPTION
**Pull request by sanderson**
Adds information about the newly available monitoring dashboard for InfluxDB Cloud Dedicated customers.

- [x] Rebased/mergeable

### CODE REVIEW

File: content/influxdb/cloud-dedicated/admin/monitor-your-cluster.md [hunk 1]
```diff
@@ -0,0 +1,383 @@
...
+
+#### Memory Usage Ingesters (k8s)
+
+Memory usage of the Ingester pod containers per cgroup as reported by Kubernetes.
+Usage is reported in a magnitude of bytes.
+The memory limit is represented by the top line in the visualization.
+
+#### CPU Utilization Routers (k8s)
+
+CPU Utilization of Ingester router pods as reported by the Kubernetes container usage.
```
Comment (jstirnaman) [inline content/influxdb/cloud-dedicated/admin/monitor-your-cluster.md:164]:
```suggestion
CPU utilization of Ingester router pods as reported by the Kubernetes container usage.
```

File: content/influxdb/cloud-dedicated/admin/monitor-your-cluster.md [hunk 2]
```diff
@@ -0,0 +1,383 @@
...
+following levels:
+
+- **L0**: Uncompacted
+- **L1**: 4 L0 files compacted together
+- **L2**: 4 L1 files compacted together
+- **L3**: 4 L2 files compacted together
+
+Parquet files store data partitioned by time and optionally tags
+_(see [Manage data partition](https://docs.influxdata.com/influxdb/cloud-dedicated/admin/custom-partitions/))_.
+Once four L0 files accumulate for a partition, they are are eligible for compaction.
```
Comment (jstirnaman) [inline content/influxdb/cloud-dedicated/admin/monitor-your-cluster.md:325]:
```suggestion
After four L0 files accumulate for a partition, they are are eligible for compaction.
```

File: content/influxdb/cloud-dedicated/admin/monitor-your-cluster.md [hunk 3]
```diff
@@ -0,0 +1,383 @@
...
+If the compactor is keeping up with the incoming write load, all compaction
+events will have exactly 4 files. If the number of L0 files compacted begins to
```
Comment (jstirnaman) [inline content/influxdb/cloud-dedicated/admin/monitor-your-cluster.md:327]:
```suggestion
events will have exactly four files. If the number of L0 files compacted begins to
```

**Review by jstirnaman — APPROVED** [review]
Thanks for doing this (I see what you meant ;)). A few minor suggestions.

File: content/influxdb/cloud-dedicated/admin/monitor-your-cluster.md [hunk 4]
```diff
@@ -0,0 +1,383 @@
...
+  Use the Grafana dashboard provided by InfluxData to monitor your
+  InfluxDB Cloud Dedicated cluster.
+menu:
+  influxdb_cloud_dedicated:
+    parent: Administer InfluxDB Cloud
+weight: 104
+---
+
+Use the Grafana dashboard provided by InfluxData to monitor your
+{{< product-name >}} cluster.
```
Comment (sanderson) [inline content/influxdb/cloud-dedicated/admin/monitor-your-cluster.md:14]:
```suggestion
{{< product-name >}} cluster.

{{% note %}}
#### Not available for all clusters

{{< product-name >}} monitoring dashboards are not available for all clusters.
For questions about availability, [contact InfluxData support](https://support.influxdata.com).
{{% /note %}}
```

**Pull request merged**